### PR TITLE
Update to .NET MAUI RC3

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
### Description of Change ###

 

 ### Linked Issues ###
 

 - Fixes #

 ### PR Checklist ###
 
 - [ ] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [ ] Has samples (if omitted, state reason in description)
 - [ ] Rebased on top of `main` at time of PR
 - [ ] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

File: samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj [hunk 1]
```diff
@@ -35,9 +35,9 @@
     <MauiImage Include="Resources\Images\*" />
     <EmbeddedResource Include="Resources\Embedded\*" />
 
-    <PackageReference Include="CommunityToolkit.Maui.Markup" Version="1.0.0-rc1" />
+    <PackageReference Include="CommunityToolkit.Maui.Markup" Version="1.0.0-rc2" />
     <PackageReference Include="CommunityToolkit.Mvvm" Version="8.0.0-preview3" />
-    <PackageReference Include="Microsoft.Extensions.Http" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="6.0.2-mauipre.1.22102.15" />
```
Comment (VladislavAntonyuk) [inline samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj:40]: it has the maui in the version name. I think it suits better

**Comment by VladislavAntonyuk** [comment]
It still installs 6.0.202 instead of 6.0.300. let's wait a few hours before it is resolved

File: samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj (hunk 1 above)
Comment (brminnick) [reply samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj:40]: Makes sense. Now that MAUI targets .NET SDK v6.0.300, I assume we'll be able to update this to Microsoft.Extensions.Http v6.0.3 once it's released

**Review by brminnick — APPROVED** [review]

**Pull request merged**